keyboards/hator/gravity/tkl/info.json

```diff
@@ -0,0 +1,304 @@
+{
```

Comment:
duplicate `info.json` and `keyboard.json` in the same directory makes no sense. Please remove this file